shepherd_tmux.sh

```diff
@@ -12,11 +12,11 @@ tmux send-keys -t 'Shepherd:0.3' 'cd shepherd' Enter
 tmux send-keys -t 'Shepherd:0.4' 'cd shepherd/sensors' Enter
 tmux send-keys -t 'Shepherd:0.5' 'cd shepherd' Enter
 
-tmux send-keys -t 'Shepherd:0.0' 'python3 Shepherd.py' Enter
+tmux send-keys -t 'Shepherd:0.0' 'python3 shepherd.py' Enter
 tmux send-keys -t 'Shepherd:0.1' 'echo free terminal' Enter
 tmux send-keys -t 'Shepherd:0.3' 'python3 server.py' Enter
 tmux send-keys -t 'Shepherd:0.4' 'make clean && make && ./dev_handler' Enter
-tmux send-keys -t 'Shepherd:0.5' 'python3 scoreboard_server.py' Enter
+tmux send-keys -t 'Shepherd:0.5' 'python3 ydl.py' Enter
 tmux send-keys -t 'Shepherd:0.2' 'sleep 20' Enter
 tmux send-keys -t 'Shepherd:0.2' 'python3 sensor_interface.py' Enter
 
```

Comment:
By the way, this script can sometimes break on docker windows machines, idk why. I have a gross but foolproof version somewhere if I should find it?

Reply:
basically just have to add no-ops

Reply:
oop we should probably figure out why
Also does this work on non-docker windows?